### PR TITLE
Add support for 2023 annual report download

### PR DESCRIPTION
**Pull request by gabalafou**

### CODE REVIEW

File: apps/labs/pages/api/annual-report.pdf.ts [hunk 1]
```diff
@@ -25,9 +25,17 @@ export default async function handler(req: NextRequest) {
...
+  switch (req.nextUrl.searchParams.get('year')) {
+    case '2022':
+      // Returning fetch comes from Next.js docs:
+      // https://nextjs.org/docs/api-routes/edge-api-routes#forwarding-headers
+      return fetch(
+        'https://a.storyblok.com/f/152463/x/20372ca74f/quansight-labs-annual-report-2022.pdf',
+      );
+    case '2023': // falls through to default
+    default:
+      // TODO: put Storyblok URL here
```
Comment (gabalafou) [inline apps/labs/pages/api/annual-report.pdf.ts:38]: Converted this PR to a draft until we have the link to the 2023 pdf to put here.

cc @Katebrack @trallard

File: apps/labs/pages/api/annual-report.pdf.ts [hunk 2]
```diff
@@ -25,9 +25,17 @@ export default async function handler(req: NextRequest) {
...
+      return fetch('https://labs.quansight.org/');
```
Comment (pavithraes) [inline apps/labs/pages/api/annual-report.pdf.ts:39]: Here's the link to the 2023 report:

```suggestion
      return fetch(
        'https://a.storyblok.com/f/152463/x/b94afd375b/labs-annual-report-2023.pdf'
      );
```

Comment (pavithraes) [inline apps/labs/pages/api/annual-report.pdf.ts:39]: Here's the link to the 2023 report:

```suggestion
      return fetch(
        'https://a.storyblok.com/f/152463/x/fb9d254ecd/quansight-labs-annual-report-2023.pdf'
      );
```

File: apps/labs/pages/api/annual-report.pdf.ts (hunk 1 above)
Comment (pavithraes) [inline apps/labs/pages/api/annual-report.pdf.ts:38]: Adding URL in https://github.com/Quansight/Quansight-website/pull/831/files#r1509120928, hence:

```suggestion
```

**Review by pavithraes — COMMENTED** [review]
Besides the 2023 report link already noted as TODO, everything looks good to me :)

**Comment by gabalafou** [comment]
Preview link is working: https://labs-ht0rj1ufz-quansight.vercel.app/annual-reports/quansight-labs-annual-report-2023.pdf

**Comment by trallard** [comment]
LGTM, shall we merge @gabalafou @pavithraes

**Review by trallard — APPROVED** [review]

**Pull request merged**